### DISCUSSION TITLE
Snapshot amount

### DISCUSSION THREAD

**Discussion by EpicLPer** [dougiteixeira/proxmoxve#290] [Feature request] [answered]
Heya,

it would be nice to see how many Snapshots a VM currently has, on top of other Feature Requests where you can directly create Snapshots for a VM.

Thanks!

**Comment by EpicLPer** [comment]
Are there by any chance updates for this? :)

**Comment by EpicLPer** [comment]
Any updates? ^^"

**Comment by mZ738** [comment, marked as answer]
5.2.0 has a **Snapshots** sensor per VM and container: the count, with the names newest first and the time of the newest as attributes. Diagnostic and off by default — enable it on the entity. (The *Create snapshot* button came with 5.1.0.) https://github.com/dougiteixeira/proxmoxve/releases/tag/5.2.0
